### ISSUE TITLE
The best blank WP theme

### ISSUE THREAD

**Issue by JohnnyWalkerDesign** [andjosh/naked-wordpress#9]
This theme is brilliant, easily the best empty theme I've come across, but I don't know why you say you're aiming it at "Designers". This is definitely useful for PHP Developers that are new to the world of WordPress. Or even those who have used WordPress a fair amount and understand it, but like their themes to be well commented (like myself). Knowing that whoever takes over after me will have a better chance of understanding what's going on is a good thing.

Basically, every WordPress theme should be this commented. I recommend this theme to everyone I can.

Keep up the good work!

**Comment by andjosh** [comment]
Thanks! You make a good point - I just labelled it as such from the outset because I was helping designer friends at the time.

**Comment by andjosh** [comment]
I should think about changing the copy, though.
